app/helpers/layout_helper.rb

```diff
@@ -21,6 +21,6 @@ def sidebar_classes
   # overwritten to return 'container-fluid' for Bootstrap full-width layout
   # @return [String]
   def container_classes
-    'container main-blacklight-container'
```

Comment:
@cdmo do we still need [this](https://github.com/psu-libraries/psulib_blacklight/blob/ab0a9cfc8a6692cd01aa933ec9b583ca178a7136/app/javascript/psulib_blacklight/styles/_mobile.scss#L34) after this change?

Reply:
Oh shoot, I meant to just remove `main-blacklight-container` thank you for catching. But, yeah, we could just kill the whole class because it's identical to it's parent.